frontend/src/pages/404Page.js

```diff
@@ -6,20 +6,42 @@ const PageNotFound = () => {
   return (
     <Container maxW="xl" centerContent>
       <Box
+        alignItems="center"
         d={"flex"}
-        justifyContent={"space-between"}
+        //  justifyContent={"space-between"}
+        flexDirection={"column"}
         w={"100%"}
         h={"100%"}
         p={"50px"}
 		fontSize={"3rem"}
-		textColor={"green"}
+		textColor={"black"}
       >
-        <h1>
+        <h1 style={{padding:"0px", }}>
           404  :) 
-		  <br/>
+          </h1>
+		  {/* <br/> */}
+      <h1>
 		  Page Not Found 
         </h1>
+        <a class="login-button" href="https://scholar-chat-orcid.herokuapp.com/">
```

Comment:
the button should send the user to the endpoint dynamically as mentioned by chirag in the comments